src/cdk-experimental/popover-edit/popover-edit.spec.ts

```diff
@@ -316,6 +341,49 @@ describe('CDK Popover Edit', () => {
           expect(component.getInput()!.value).toBe('Hydrogen');
         }));
 
+        it('positions the lens at the top left corner and spans the full width of the cell',
+           fakeAsync(() => {
+             component.openLens();
+
+             const paneRect = component.getEditPane()!.getBoundingClientRect();
+             const cellRect = component.getEditCell().getBoundingClientRect();
+
+             expect(paneRect.width).toEqual(cellRect.width);
```

Comment:
In general we prefer to use `toBe` rather than `toEqual`. `toEqual` might let some errors through due to the type coercion.